addons/sale_timesheet/models/project_update.py

```diff
@@ -38,7 +38,7 @@ def _get_services_values(self, project):
                 ('is_downpayment', '=', False),
             ]),
         )
-        name_by_sol = dict(sols.with_context(with_price_unit=True).name_get())
+        name_by_sol = dict(sols.name_get())
```

Comment:
I would keep the context key and add it in the action related to the grid. And more in our apps if needed. See with @aju-odoo. Removing the condition will displaying like that everywhere in odoo (all apps).
```suggestion
        name_by_sol = dict(sols.with_context(with_price_unit=True).name_get())
```